forge-gui-mobile/src/forge/itemmanager/views/ItemListView.java

```diff
@@ -154,9 +155,7 @@ public void zoomSelected() {
     @Override
     public void selectAll() {
         selectedIndices.clear();
-        for (Integer i = 0; i < getCount(); i++) {
-            selectedIndices.add(i);
-        }
+        IntStream.range(0, getCount()).forEach(i -> selectedIndices.add(i));
```

Comment:
doesn't `selectedIndices::add` would work too like in the file before?

Reply:
Took me a moment to figure out why the IDE did one one way and one the other. `selectedIndicies` is declared as final in ImageView but not here in ItemListView. The IDE was apparently accounting for the possibility that the ArrayList `selectedIndicies` points to could change in the middle of a loop. The lambda would account for that, whereas the method reference would resolve the reference once, then call `add` on the same object until the loop's conclusion even if the field changed.

Regardless, there isn't actually any code that would ever replace `selectedIndicies`, let alone in the middle of the select all loop, so I've changed it to final and updated it here.